PSCompatibilityCollector/Microsoft.PowerShell.CrossCompatibility/Collection/PlatformInformationCollector.cs

```diff
@@ -22,13 +23,74 @@ namespace Microsoft.PowerShell.CrossCompatibility.Collection
     /// </summary>
     public class PlatformInformationCollector : IDisposable
     {
+        /// <summary>
+        /// Collect all release info files into a lookup table in memory.
+        /// Overrides pre-existing keys if there are duplicates.
+        /// </summary>
+        /// <returns>A dictionary with the keys and values of all the release info files on the machine.</returns>
+        public static IReadOnlyDictionary<string, string> GetLinuxReleaseInfo()
+        {
+            var dict = new Dictionary<string, string>();
```

Comment:
Maybe a more descriptive name could be useful? What about creating it in a case insensitive way?

Reply:
A more descriptive name for the method? Not sure what to call it other than this, but open to suggestions.

I thought about case-sensitivity, but ultimately this is Linux-specific and there's nothing to stop two keys being added that differ only by case

Reply:
No, for `dict`

Reply:
Oh! Got it :)